packages/mobile/src/home/WalletHome.tsx

```diff
@@ -110,10 +101,17 @@ function WalletHome() {
   }
 
   const shouldShowCashInBottomSheet = () => {
-    const hasStable = STABLE_CURRENCIES.some((currency) =>
-      balances[currency]?.isGreaterThan(STABLE_TRANSACTION_MIN_AMOUNT)
+    // If there are no core tokens then we are either still loading or loading failed.
+    if (coreTokenBalances.length < 2) {
+      return false
+    }
+    const hasStable = !!coreTokenBalances.find(
+      (token) => token.balance.gte(STABLE_TRANSACTION_MIN_AMOUNT) && token.address !== celoAddress
     )
-    const hasCelo = balances[Currency.Celo]?.isGreaterThan(CELO_TRANSACTION_MIN_AMOUNT)
+
+    const hasCelo = coreTokenBalances
+      .find((token) => token.address !== celoAddress)
```

Comment:
should this condition be `token.address === celoAddress` instead?

Reply:
Yes! 🤦‍♂️ 
Thanks!